docker-compose.deploy.yml

```diff
@@ -31,7 +31,11 @@ services:
           - node.role == manager
     networks:
       - overlay_net
-
+    logging:
```

Comment:
I configured logging to go to Logstash from the rest of the services as well, so the need to using `docker service logs` would stay minimal